week1/project/ecommerce/src/App.jsx

```diff
@@ -0,0 +1,41 @@
+import './App.css';
+import { useState } from 'react';
+import allCategories from './fake-data/all-categories.js';
+import products from './fake-data/all-products.js';
+import Header from './components/Header.jsx';
+import Categories from './components/Categories.jsx';
+import ProductList from './components/ProductList.jsx';
+
+export default function App() {
+    const [selectedCategory, setSelectedCategory] = useState(null);
+
+    const handleCategorySelect = (category) => {
+        setSelectedCategory(category);
+    };
+
+    const normalizeCategory = (category) => {
+        return category.replace('FAKE:', '').trim();
```

Comment:
```suggestion
        return category.replace('FAKE: ', '');
```
**Suggestion:**
you don't need trim